### PR TITLE
Eobs 1879 full blood glucose observation

### PR DESCRIPTION
**Pull request by Nayira**
# Features Added/Changed:
submit_full_blood_glucose_obs
submit_full_blood_product_obs
submit_full_height_obs
submit_full_neurological_obs
submit_full_weight_obs

# Feature file checks:
- [x] JIRA ticket reference number
- [x] Scenarios covered listed

# Possible issues:
There is some commented out code
-  submit_full_weight_obs - for future implementation

# General Checks:
- [x] Linting passing with score 10/10 (Happy Travis!)
- [x] All Tests passing on local run
- [x] All commented out code has a reason for it documented
- [x] All methods have a detailed docstring / changed methods have updated docstring
- [x] Documentation up to date
- [x] Timings on expected conditions reviewed
- [x] Deprecated selectors reviewed
- [x] Code re-usability reviewed

### CODE REVIEW

File: features/submit_full_blood_glucose_obs.feature [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  Scenario Outline: Full Blood Glucose Observation is correctly submitted.
+    Given a user with the Nurse role logs into the app
+    And they view the My Patients list
+    And the My Patients list has loaded
+    When Patient Doyle, Worth Scott is selected
+    And the Take observation button is selected
+    And the Blood Glucose observation is selected from the list
+    Then the Blood Glucose observation form is displayed
+    When the value <vol_blood_glu> is inputted in the Blood Glucose (mmol/L) field
```
Comment (Gimpneek) [inline features/submit_full_blood_glucose_obs.feature]: inputted -> input

File: features/submit_full_height_obs.feature [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  Scenario Outline: Height observation is correctly submitted
+    Given a user with the Nurse role logs into the app
+    And they view the My Patients list
+    And the My Patients list has loaded
+    When Patient Doyle, Worth Scott is selected
+    And the Take observation button is selected
+    And the Height observation is selected from the list
+    Then the Height observation form is displayed
+    When the value <height> is inputted in the Height (m) field
```
Comment (Gimpneek) [inline features/submit_full_height_obs.feature]: inputted -> input

File: features/submit_full_blood_glucose_obs.feature (hunk 1 above)
Comment (Nayira) [reply features/submit_full_blood_glucose_obs.feature]: Will refactor with EOBS-1889_partial_reason_popup as other changes will be needed in the gherkin due to implementation of regex.

File: features/submit_full_height_obs.feature (hunk 2 above)
Comment (Nayira) [reply features/submit_full_height_obs.feature]: Will refactor with EOBS-1889_partial_reason_popup as other changes will be needed in the gherkin due to implementation of regex.

**Pull request merged**